ppa/editorial/templates/editorial/snippets/attribution.html

```diff
@@ -0,0 +1,10 @@
+{# Snippet to include authors and date attribution to editorial posts #}
+{% for block in page.authors %}
+    {% if block.value.url and add_link %}
```

Comment:
Could do what we've done elsewhere when we sometimes have a link and sometimes not: https://github.com/Princeton-CDH/derrida-django/blob/master/derrida/books/templates/components/books-list-item.html#L25-L26 

I think that might be a bit better than this.

Reply:
I had to do a hybrid of both approaches. The issue is that there's not a great way to use `yesno` and also have a switch for "never include" links. When I push I think what I did will be clearer and more explicit (if a bit more verbose in the name of template clarity)